ygot/diff_test.go

```diff
@@ -838,6 +838,16 @@ func TestDiff(t *testing.T) {
 				Val: &gnmipb.TypedValue{Value: &gnmipb.TypedValue_StringVal{"cabernet-sauvignon"}},
 			}},
 		},
+	}, {
+		desc:   "single path addition in modified with OriginalPathsOnly set",
+		inOrig: &renderExample{},
+		inMod: &renderExample{
+			Str: String("cabernet-sauvignon"),
+		},
+		inOpts: []DiffOpt{&DiffPathOpt{
+			OriginalPathsOnly: true,
+		}},
+		want: &gnmipb.Notification{},
```

Comment:
Could you add some fields for modify & delete to make sure that the behaviour for those are unaffected by this flag?

Reply:
Done.